tools/RAiDER/aria/prepFromARIA.py

```diff
@@ -0,0 +1,212 @@
+def get_bbox_GUNW(f:str, buff:float=1e-5):
+    """ Get the bounding box (SNWE) from an ARIA GUNW product """
+    import shapely.wkt
+    ds       = xr.open_dataset(f)
+    poly_str = ds['productBoundingBox'].data[0].decode('utf-8')
+    poly     = shapely.wkt.loads(poly_str)
+    W, S, E, N = poly.bounds
+
+    ### buffer slightly?
+    W, S, E, N = W-buff, S-buff, E+buff, N+buff
+    return [S, N, W, E]
+
+
+def parse_dates_GUNW(f:str):
+    """ Get the ref/sec date from the filename """
+    sec, ref = f.split('-')[6].split('_')
+
+    return int(sec), int(ref)
+
+
+def parse_time_GUNW(f:str):
+    """ Get the center time of the secondary date from the filename """
+    tt = f.split('-')[7]
+    return f'{tt[:2]}:{tt[2:4]}:{tt[4:]}'
+
+
+def parse_look_dir(f:str):
+    look_dir = f.split('-')[3].lower()
+    return 'right' if look_dir == 'r' else 'left'
+
+
+def update_yaml(dct_cfg, dst='GUNW.yaml'):
+    """ Write a temporary yaml file with the new 'value' for 'key', preserving parms in example_yaml"""
+
+    template_file = os.path.join(
+                    os.path.dirname(RAiDER.__file__), 'cli', 'raider.yaml')
+
+    with open(template_file, 'r') as f:
+        try:
+            params = yaml.safe_load(f)
+        except yaml.YAMLError as exc:
+            print(exc)
+            raise ValueError(f'Something is wrong with the yaml file {example_yaml}')
+
+    params = {**params, **dct_cfg}
+
+    with open(dst, 'w') as fh:
+        yaml.safe_dump(params, fh,  default_flow_style=False)
+
+    logger.info (f'Wrote new cfg file: %s', dst)
+    return dst
+
+
+def main(args):
+    '''
+    A command-line utility to convert ARIA standard product outputs from ARIA-tools to
+    RAiDER-compatible format
+    '''
+
+    for f in args.files:
+        # version  = xr.open_dataset(f).attrs['version'] # not used yet
+        # SNWE     = get_bbox_GUNW(f)
+
+        work_dir = args.output_directory
+        # wavelen  = xr.open_dataset(f, group='science/radarMetaData')['wavelength'].item()
+        dates0   = parse_dates_GUNW(f)
+        time     = parse_time_GUNW(f)
+        heights  = getHeights(f)
+        lookdir  = parse_look_dir(f)
+
+        # makeLOSFile(f, args.los_file)
+        f_lats, f_lons = makeLatLonGrid(f, work_dir)
+        orbits     = getOrbitFile(f, work_dir)
+        # bbox_la = '37.129123314154995 37.9307480710763 -118.44814585278701 -115.494195892019'
+
+        # temp
+        dates = dates0[1]
+        orbits = orbits[0]
+
+        cfg  = {
+               'look_dir':  lookdir,
+               'weather_model': args.model,
+               'aoi_group' : {'lat_file': f_lats, 'lon_file': f_lons},
+               'date_group': {'date_list': str(dates)},
+               'height_group': {'dem': args.dem},
+               'time_group': {'time': time},
+               'los_group' : {'ray_trace': True,
+                              'orbit_file': orbits,
+                              'los_convention': 'isce',
```

Comment:
Do you need LOS convention when you are using an orbit file?

Reply:
`los_convention` shouldn't be needed with orbit.